cli_flags.go

```diff
@@ -67,6 +68,7 @@ var (
 		"Valid values are in the range [1..%d], and 0 to disable off-cpu profiling."+
 		"Default is %d.",
 		support.OffCPUThresholdMax, defaultOffCPUThreshold)
+	envVarsHelp = "Comma separated list of environment variables that should be reported with the captured profiling samples"
```

Comment:
@RononDex It's easier on reviewers if you accept review suggestions through the GH interface instead as it:

1. Reduces mental burden / review effort since reviewers know that the suggestion was committed as-is and they don't need to check again.
2. Cleaner timeline of review comments, automatic resolution.
3. Results in better code as it reduces probability of typos / bugs due to miscommunication to slip through.
4. Gives people credit for their contributions (alternative is to keep track of and remember to manually insert "Co-authored-by: XXX" statements in the final merge commit message.

```suggestion
	envVarsHelp = "Comma separated list of environment variables that should be reported with the captured profiling samples."
```